### PR TITLE
fix: modify category logic in board and post

### PR DESCRIPTION
**Pull request by stat-kwon**
### Category
- [ ] New feature
- [ ] Bug fix
- [ ] Improvement
- [x] Refactor
- [ ] etc

### Description
Post is created even without categories.

### CODE REVIEW

File: src/spaceone/board/model/post_model.py [hunk 1]
```diff
@@ -6,7 +6,7 @@
 class Post(MongoModel):
     board_id = StringField(max_length=40)
     post_id = StringField(max_length=40, generate_id='post', unique=True)
-    category = StringField()
+    category = StringField(null=True)
```
Comment (whdalsrnt) [inline src/spaceone/board/model/post_model.py:9]: Change the options as follows:
`null=True, defualt=None`

File: src/spaceone/board/service/post_service.py [hunk 2]
```diff
@@ -56,7 +56,7 @@ def create(self, params):
         board_vo = self.board_mgr.get_board(board_id)
         categories = board_vo.categories
 
-        if category := params.get('category'):
+        if category := params.get('category', None):
```
Comment (whdalsrnt) [inline src/spaceone/board/service/post_service.py:59]: Remove `None` because `<dict>.get` defaults to `None`

**Review by whdalsrnt — CHANGES_REQUESTED** [review]
code change is required.

**Comment by stat-kwon** [comment]
I have edited the part you mentioned and committed it. @whdalsrnt

**Pull request merged**